src/AppLayout.tsx

```diff
@@ -54,7 +55,11 @@ import pncLogoText from './pnc-logo-text.svg';
 export const AppLayout = () => {
   const webConfig = webConfigService.getWebConfig();
 
-  const user = keycloakService.isKeycloakAvailable ? keycloakService.getUser() : null;
+  const [user, setUser] = useState(keycloakService.isKeycloakAvailable ? keycloakService.getUser() : null);
+
+  authBroadcastService.getChannel().onmessage = (event: MessageEvent<IAuthBroadcastMessage>) => {
```

Comment:
I think some technical details could be encapsulated a little bit. 

`authBroadcastService.getChannel().onmessage` can be converted to something like:

1) `authBroadcastService.onMessage((event) => { setUser(..) });` 

or even better:

2) `authBroadcastService.addMessageListener((event) => { setUser(..) });` which provides better options when some additional global actions like message parsing etc. are required.